brainscore_language/benchmarks/syntaxgym/test_suites.py

```diff
@@ -0,0 +1,108 @@
+suite11 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/mvrr.json'
+suite12 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/mvrr_mod.json'
+suite13 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/nn-nv-rpl.json'
+suite14 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npi_orc_any.json'
+suite15 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npi_orc_ever.json'
+suite16 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npi_src_any.json'
+suite17 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npi_src_ever.json'
+suite18 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npz_ambig.json'
+suite19 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npz_ambig_mod.json'
+suite20 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npz_obj.json'
+suite21 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/npz_obj_mod.json'
+suite22 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/number_orc.json'
+suite23 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/number_prep.json'
+suite24 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/number_src.json'
+suite25 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_orc_fem.json'
+suite26 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_orc_masc.json'
+suite27 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_prep_fem.json'
+suite28 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_prep_masc.json'
+suite29 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_src_fem.json'
+suite30 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/reflexive_src_masc.json'
+suite31 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/subordination.json'
+suite32 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/subordination_orc-orc.json'
+suite33 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/subordination_pp-pp.json'
+suite34 = 'https://raw.githubusercontent.com/cpllab/syntactic-generalization/nextflow/test_suites/json/subordination_src-src.json'
```

Comment:
Can we index these by commit so that we always reference the same files, even if commits are made?
E.g. (this is a slightly different url, but there should be a way to get the commit hash in there; on github by simply pressing `y`):
```
https://github.com/cpllab/syntactic-generalization/raw/43f1f86b746254406dafc89bd17b9d4976607ef6/test_suites/json/center_embed.json
```